### PR TITLE
Adds Stat.none

### PR DESCRIPTION
**Pull request by younggam**
Stat.none is for customized info table.
input:    <--this part would be confusing
bla-table-bla-table
bla-table-bla-table
bla-table-bla-table

### CODE REVIEW

**Comment by Quezler** [comment]
as per your example would it not still show the `:` part?

**Comment by younggam** [comment]
ohno you are genius. which method can be alternative?
`info:`
makes sense?

File: core/assets/bundles/bundle.properties [hunk 1]
```diff
@@ -649,6 +649,7 @@ stat.minetier = Mine Tier
 stat.payloadcapacity = Payload Capacity
 stat.commandlimit = Command Limit
 stat.abilities = Abilities
+stat.none =
```
Comment (summetdev) [inline core/assets/bundles/bundle.properties:652]: Put an empty key to each bundle is pretty strange.

Comment (younggam) [reply core/assets/bundles/bundle.properties:652]: it's valid for .propeties

Comment (summetdev) [reply core/assets/bundles/bundle.properties:652]: Why do you even need an *empty* bundle key, bundles are for translations.

Comment (younggam) [reply core/assets/bundles/bundle.properties:652]: becuz Stat.bla matches with bundle
if there is no bundle for it, it will show ???stat.none???

Comment (genNAowl) [reply core/assets/bundles/bundle.properties:652]: or you can hardcode it into the display to make it just display nothing

Comment (younggam) [reply core/assets/bundles/bundle.properties:652]: hmm.. ok

**Comment by younggam** [comment]
currently, if Stat.none, nothing will be shown except StatValue

**Comment by Quezler** [comment]
👀

**Pull request closed without merging**